src/api/app/controllers/webui/projects/label_templates_controller.rb

```diff
@@ -22,8 +22,10 @@ def create
 
         if @label_template.save
           redirect_to project_label_templates_path(@project)
+          flash[:success] = 'Label template created successfully'
         else
           render :new
+          flash[:error] = 'Failed to create the label template'
```

Comment:
Why and what can I do about it?

Reply:
You'll get a validation message when presented with the form again
![image](https://github.com/user-attachments/assets/364b60b8-fe49-45c1-b932-0c52fdf9302e)

Reply:
Why can I even press create without filling out this field? :)

Reply:
Hm, good point

Reply:
Alright, I leveraged the native html form required attribute to prevent the form from being submitted if the fields are empty